src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java

```diff
@@ -89,18 +91,22 @@ public void we_send_the_to_eiffel_intelligence_for_creation(String subscriptionN
         assertEquals("Failed to create subscription. Response: " + response.getBody(), 200, response.getStatusCode());
     }
 
-    @Given("^all previous tests passes\\.$")
-    public void all_previous_tests_passes() {
-        //Just for cucumber to make sence
-    }
-
-    @When("^future story is done\\.$")
-    public void future_story_is_done() {
+    @Given("^the jenkins job \"([^\"]*)\" is triggered$")
+    public void the_jenkins_job_is_triggered(String jenkinsJobToTrigger) throws Throwable {
+        Map<String, String> parameters = new HashMap<String, String>();
+        parameters.put("EVENT_ID", "");
+        parameters.put("TEST_CASE_NAME", "");
 
+        StepsUtils.triggerJenkinsJobWithParameters(jenkinsJobToTrigger, parameters, JENKINS_TOKEN);
     }
 
-    @Then("^everything should be fine\\.$")
-    public void everything_should_be_fine() {
+    @When("^all jenkins jobs has been triggered$")
+    public void the_jenkins_job_has_been_triggered() throws Throwable {
+        StepsUtils.hasJenkinsJobsBeenTriggered(jenkinsJobNames);
+    }
 
+    @Then("^the test was a succcess$")
+    public void the_test_was_a_succcess() {
+        //Just for cucumber to make sence
```

Comment:
We have an open discussion about this in a different Pull-request.

Reply:
Ye, i'll wait until we have decided :)